nautobot_golden_config/tests/test_views.py

```diff
@@ -151,7 +151,7 @@ def test_configreplace_export(self):
         self.assertEqual(response.headers["Content-Type"], "text/csv")
         last_entry = models.ConfigReplace.objects.last()
         csv_data = response.content.decode().splitlines()
-        expected_last_entry = f"{last_entry.name},{last_entry.platform.id},{last_entry.description},{last_entry.regex},{last_entry.replace}"
+        expected_last_entry = f"{last_entry.name},{last_entry.platform.slug},{last_entry.description},{last_entry.regex},{last_entry.replace}"
```

Comment:
Not fully understanding why this would change?

Reply:
Makes sense, was looking for the URL that mapped to that, but forget that is abstracted now...